test/src/kava.spec.js

```diff
@@ -432,6 +477,8 @@ describe('KavaPlugin', function() {
     });
 
     it('should send VIEW event', done => {
+      sandbox.stub(window.navigator.connection, 'effectiveType').value('2g');
```

Comment:
we are missing tests for conditional data, what if connection doesn't exist?

Reply:
So you want me to add a test and force connection to be undefined?